src/avvoenv/source/vault.rs

```diff
@@ -75,6 +80,14 @@ impl Client {
         Ok(())
     }
 
+    pub fn app_id_auth(&mut self, app_id: String, user_id: String) -> Result<(), errors::Error> {
+        self.resolve_leader()?;
```

Comment:
Can you test if this is required?

I'm pretty sure it's a bug in the old versions of Vault, but I don't remember if it's all auth or just ldap.

If it is required could you leave a comment similar to the one in the ldap_auth function.

Reply:
Not required. Removing this.